semgrep/semgrep/util.py

```diff
@@ -174,3 +179,37 @@ def compute_semgrep_path() -> str:
 
 def compute_spacegrep_path() -> str:
     return compute_executable_path("spacegrep")
+
+
+def liststartswith(l: List[Any], head: List[Any]) -> bool:
```

Comment:
Does List[str] not work?

Reply:
`str` would work, but this function also works with list's containing anything that defines `==`, not just strings.

Reply:
Maybe `T` makes more sense?